src/onboarding/Start.js

```diff
@@ -191,40 +205,46 @@ class StartContent extends React.PureComponent {
         </NetworkChooser>
 
         <TwoActions>
-          <Action>
-            <p>
-              <Text size="large" color={theme.textSecondary}>
-                Then create a new organization
-              </Text>
-            </p>
-            <Button
-              mode="strong"
-              onClick={this.props.onCreate}
-              disabled={!canCreate}
-            >
-              Create a new organization
-            </Button>
-            {this.renderWarning()}
-          </Action>
+          {!responsiveLanding && (
+            <Action>
+              <p>
+                <Text size="large" color={theme.textSecondary}>
+                  Then create a new organization
+                </Text>
+              </p>
+              <Button
+                mode="strong"
+                onClick={this.props.onCreate}
+                disabled={!canCreate}
+              >
+                Create a new organization
+              </Button>
+              {this.renderWarning()}
+            </Action>
+          )}
           <form onSubmit={onOpenOrganization}>
             <Action>
               <p>
                 <Text size="large" color={theme.textSecondary}>
-                  Or open an existing organization
+                  {responsiveLanding
+                    ? 'Enter your organization name'
+                    : 'Or open an existing organization'}
                 </Text>
               </p>
 
               <OpenOrganization>
                 <Field>
-                  <TextInput
+                  <StyledTextInput
                     id="onboard-start-domain"
-                    style={{ textAlign: 'right' }}
                     onChange={onDomainChange}
                     value={domain}
+                    placeholder="Organization name"
                   />
-                  <label htmlFor="onboard-start-domain">
-                    <Text weight="bold"> .aragonid.eth</Text>
-                  </label>
+                  <div>
+                    <label htmlFor="onboard-start-domain">
+                      <Text weight="bold"> .aragonid.eth</Text>
```

Comment:
This should be `18px` on small screens, but since this size doesn’t exist, maybe we could use [`16px` (`large`)](https://github.com/aragon/aragon-ui/blob/master/src/utils/styles/font.js#L6) instead? Or should we set it manually only here? Or we could add another size… 😅 /cc @jounih

Reply:
large works :)